src/main/drivers/pwm_output.c

```diff
@@ -766,4 +769,10 @@ void beeperPwmInit(const ioTag_t tag, uint16_t frequency)
         beeperPwm.enabled = false;
     }
 }
+#endif // USE_BEEPER
+#else // USE_PWM_OUTPUT
+bool isMotorProtocolDshot(void)
```

Comment:
Shouldn't be needed - code blocks that check on this should be made conditional to `USE_PWM_OUTPUT`, since they won't be needed without PWM output (this probably applies to pretty much all of `mixer.c`.